src/tools/gxpkg.ss

```diff
@@ -316,7 +316,7 @@
                 (error "Cannot build unknown package" pkg)))
            (build.ss (pkg-build-script pkg)))
       (displayln "... build " pkg)
-      (run-process [build.ss "deps"]
+      #;(run-process [build.ss "deps"] ;; TODO: Should we delete this, or keep it, or what?
```

Comment:
I think we want to check the meta in case it supports deps, and if so call it.

Reply:
Actually, I saw that no one in `gerbil-directory` uses `deps`. Can we just remove it?

Reply:
alright, let's kill it then. 
Let's also update the guide for gxpkg, I think it mentions it there.

Reply:
It's already removed from `doc/guide/build.md`, but I see you probably mean `doc/guide/package-manager.md`. Removed it there, too. Thanks. Pushed.